src/components/task-plan/homework/exercises.cjsx

```diff
@@ -183,6 +183,11 @@ ExerciseTable = React.createClass
   renderExerciseRow: (exerciseId, index, hasTeks) ->
     {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
     content = ExerciseStore.getContent(exerciseId)
+    contentDiv = document.createElement('div')
+    contentDiv.innerHTML = content
+    
+    if contentDiv.children.length > 1
```

Comment:
We could use:
```
table.exercise-table {  
  td {
    * + * {
      display: none;
    }
  }
}
```

The * + * will select any elements except the first regardless of type.  Tested it on chrome, ie, firefox.